### ISSUE TITLE
Add optional attribute name parameter to sorting service object

### ISSUE THREAD

**Issue by rlgreen91** [EBWiki/EBWiki#1981]
Update the service object `SortCollectionOrdinally` to take an optional parameter `column_name`.  This parameter should default to `name`, and whatever value is provided should be used when sorting the collection.  For example, if the value is `name`, then you should sort the collection ordinally based on the name attribute of the corresponding model.  Note that you will also have to pass along the collection as a keyword argument as well - just name it `collection`.

You can test this using the Gender model, which instead of a name has a sex.  

Then, in the controllers, update the lines using `SortCollectionOrdinally` to pass the parameters along properly.  Remember that when using the service object for genders, you'll need to set the `column_name` parameter as sex.

**Comment by trystant** [comment]
The `SortCollectionOrdinally` was passed a `Gender` model, so I have a patch that creates an alias column `:name` so that Gender works in #2028

**Comment by StlMaris123** [comment]
Hi @trystant , should this issue be closed?

**Comment by rlgreen91** [comment]
No, go ahead and make the changes.  We might need this for other models in the future, and adding a bunch of aliases isn't practical.
